apps/web/src/hooks/useNotificationSSE.ts

```
@@ -0,0 +1,140 @@
+          }
+          throw new Error(`SSE open failed: ${response.status}`);
+        },
+
+        onmessage: event => {
+          if (event.event === 'notification') {
+            try {
+              const payload = JSON.parse(event.data) as NotificationSsePayloadT;
+              const deepLink = resolveDeepLink(payload);
+              const action = deepLink
+                ? { label: '바로가기', onClick: () => router.push(deepLink) }
+                : undefined;
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)